docs/pages/operations-guide.md

```diff
@@ -10,123 +10,149 @@ redirect_from: "/ops-guide"
+- `svc.yml` template.
+- New environment variables.
+- Custom logic implemented by the service developer (if any).
 
 In addition, the Scheduler now has a fourth piece:
 - Preexisting state in Zookeeper
 
-As such, Scheduler reconfiguration is slightly different from initial deployment, as the Scheduler is now comparing its current state to a non-empty prior state and determining what needs to be changed.
+Scheduler reconfiguration is slightly different from initial deployment because the Scheduler is now comparing its current state to a non-empty prior state and determining what needs to be changed.
 
-1. After the Scheduler has rendered its `svc.yml` against the new environment variables, it has two Service Specs, reflecting two different configurations:
-    1. The Service Spec that was just rendered, reflecting the config change.
-    1. The prior Service Spec (or "Target Configuration") which was previously stored in ZK.
-1. The Scheduler will automatically compare the changes between the old and new Service Specs:
-    1. __Change validation__: Certain changes such as editing volumes and scale-down are not currently supported, as they are both complicated and dangerous to get wrong.
-        - If an invalid change is detected, the scheduler will emit an error message and refuse to proceed until the user has reverted the change by relaunching the Scheduler app in Marathon with the prior config.
-        - Otherwise, the new configuration is stored in ZK as the new Target Configuration and the change deployment proceeds as described below.
-    1. __Change deployment__: The Scheduler effectively produces a `diff` between the current state and some future state, including all of the Mesos calls (reserve, unreserve, launch, destroy, etc...) needed to get there. For example, if the number of tasks has been increased, then the Scheduler will launch the correct number of new tasks. If a task configuration setting has been changed, the Scheduler will deploy that change to the relevant affected tasks by relaunching them. Tasks which aren't affected by the configuration change will be left as-is.
+1. After the Scheduler has rendered its `svc.yml` against the new environment variables, it has two Service Specs, reflecting two different configurations.
+    1. The Service Spec that was just rendered, reflecting the configuration change.
+    1. The prior Service Spec (or "Target Configuration") that was previously stored in ZK.
+1. The Scheduler automatically compares the changes between the old and new Service Specs.
+    1. __Change validation__: Certain changes, such as editing volumes and scale-down, are not currently supported because they are complicated and dangerous to get wrong.
+        - If an invalid change is detected, the Scheduler will send an error message and refuse to proceed until the user has reverted the change by relaunching the Scheduler app in Marathon with the prior config. <!-- or a different, correct, configuration? -->
```

Comment:
Re comment: yes

Could phrase it as: "until the user has resolved the issue by relaunching the Scheduler app in Marathon with the prior config, or with another config that passes the change validation"